tests/functional/disaster-recovery/regional-dr/test_rdr_monitoring_dashboard_ui.py

```diff
@@ -0,0 +1,256 @@
+import logging
+import pytest
+
+from time import sleep
+
+from ocs_ci.framework import config
+from ocs_ci.framework.testlib import skipif_ocs_version, tier1
+from ocs_ci.framework.pytest_customization.marks import (
+    turquoise_squad,
+    rdr_ui_failover_config_required,
+)
+from ocs_ci.helpers import dr_helpers
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.acm.acm import AcmAddClusters
+from ocs_ci.helpers.dr_helpers_ui import (
+    check_cluster_status_on_acm_console,
+    failover_relocate_ui,
+    verify_drpolicy_ui,
+    check_cluster_operator_status,
+    application_count_on_ui,
+    health_and_peer_connection_check_on_ui,
+    check_apps_running_on_selected_cluster,
+    clusters_in_dr_relationship,
+    protected_volume_count_per_cluster,
+)
+from ocs_ci.ocs.node import get_node_objs, wait_for_nodes_status
+from ocs_ci.ocs.resources.pod import wait_for_pods_to_be_running
+from ocs_ci.ocs.ui.validation_ui import ValidationUI
+from ocs_ci.ocs.ui.views import locators
+from ocs_ci.ocs.utils import enable_mco_console_plugin
+from ocs_ci.utility.utils import ceph_health_check, get_ocp_version
+
+logger = logging.getLogger(__name__)
+
+
+@tier1
+@turquoise_squad
+@rdr_ui_failover_config_required
+@skipif_ocs_version("<4.16")
+class TestRDRMonitoringDashboardUI:
+    """
+    Test class for RDR monitoring dashboard validation
+
+    """
+
+    @pytest.mark.polarion_id("OCS-5013")
+    def test_rdr_monitoring_dashboard_ui(
+        self, setup_acm_ui, dr_workload, nodes_multicluster, node_restart_teardown
+    ):
+        """
+        Test to verify the presence of RDR monitoring dashboard, various workloads
+        and their count, Cluster and Operator health status on it
+
+        """
+
+        # Enable MCO console plugin needed for DR dashboard
+        enable_mco_console_plugin()
+
+        rdr_workload = dr_workload(num_of_subscription=1, num_of_appset=1)
+        rdr_workload_count = len(rdr_workload)
+
+        dr_helpers.set_current_primary_cluster_context(
+            rdr_workload[0].workload_namespace
+        )
+
+        scheduling_interval = dr_helpers.get_scheduling_interval(
+            rdr_workload[0].workload_namespace
+        )
+        wait_time = 2 * scheduling_interval  # Time in minutes
```

Comment:
If possible, let's reduce to 1.5 times the scheduling_interval. You can do that in a new PR in case 1.5 is sufficient

Reply:
sure